toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -550,6 +546,27 @@ os:
     corruptionOption: panic
 ```
 
+The Verity-enabled root filesystem is always mounted as read-only. Its root hash
```

Comment:
I would move all of this prose to the `verity.md` file. And then in this file, only say something like: `There are multiple ways to configure a verity enabled image. For recommendations see, [Verity Image Recommendations](./verity.md)`.